#tantalum/js/calc.js

```diff
@@ -162,8 +163,10 @@ function calc() {
     formula = "error"
   }
 
+let formulatwo = formula.toFixed(0)
```

Comment:
Using `formulatwo` is kind off sloppy...
I suggest using the variable `errors` or `error` here and for the output using the variable `output`.